### PR TITLE
Adds Card user profile BB

### PR DESCRIPTION
**Pull request by rafibomb**

### CODE REVIEW

File: src/building-blocks/card-profile-stats/card-profile-stats.scss [hunk 1]
```diff
@@ -1,5 +1,3 @@
-
-
 .card-profile-stats {
```
Comment (canterberrie) [inline src/building-blocks/card-profile-stats/card-profile-stats.scss]: What's this?

Comment (rafibomb) [inline src/building-blocks/card-profile-stats/card-profile-stats.scss]: space

Comment (canterberrie) [inline src/building-blocks/card-profile-stats/card-profile-stats.scss]: ha ha

**Comment by canterberrie** [comment]
Cool BB! Merging

File: src/building-blocks/card-profile-stats/card-profile-stats.scss (hunk 1 above)
Comment (IamManchanda) [inline src/building-blocks/card-profile-stats/card-profile-stats.scss]: Lol

**Pull request merged**